### DISCUSSION TITLE
MOSAIC simulation error

### DISCUSSION THREAD

**Discussion by ShrenikaSrinivas** [eclipse/mosaic#127] [Q&A]
Hello, I have an error when I simulate my scenario. The mapping file and other files are created and placed properly. I am attaching the Mosaic log below. Would be of great help to know what is causing this issue.

[Mosaic_error_log.txt](https://github.com/eclipse/mosaic/files/6715460/Mosaic_error_log.txt)

**Comment by kschrab** [comment]
That's a weird one.. I guess  we would need morge information about your scenario (at least `scenario_config.json`, `mapping_config.json`)

**Reply by ShrenikaSrinivas** [reply]
I am attaching the scenario_config.json , mosaic log file(and also I have found another error) and mapping log file

[MOSAIC.log](https://github.com/eclipse/mosaic/files/6735123/MOSAIC.log)
[Mapping.log](https://github.com/eclipse/mosaic/files/6735124/Mapping.log)
[scenario_config.txt](https://github.com/eclipse/mosaic/files/6735163/scenario_config.txt)

**Reply by kschrab** [reply]
Now there seems to be a different error (`Unknown route 2 for vehicle with departure time 1000000000`). Is the route with id `2` present in the *.db or the *.rou.xml of your scenario?

**Reply by ShrenikaSrinivas** [reply]
Yes,the route id is there in the rou.xml file

**Reply by kschrab** [reply]
I guess you need to send us the whole scenario for further help. If you don't want to post it here, you can send it to mosaic@fokus.fraunhofer.de instead
